development/kops/cluster_wait.sh

```diff
@@ -42,21 +42,18 @@ function k {
 }
 
 # Add IAM configmap
-COUNT=0
 echo 'Waiting for cluster to come up...'
 k apply -f ./${KOPS_CLUSTER_NAME}/aws-iam-authenticator.yaml
 
 
-# In kops 1-22 metrics was updated and the port was changed to 443 from 4443. In the verison of metrics server we ship for kube versions < 1-22, it does not support binding to 443
-# patching back to old port and behavior
-if [ "${RELEASE_BRANCH}" != "1-22" ]; then
+# TODO: Delete when discontinue support for 1-18
+if [ "${RELEASE_BRANCH}" == "1-18" ]; then
     PATCH='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--secure-port=4443" },{"op": "replace", "path": "/spec/template/spec/containers/0/ports/0/containerPort", "value": 4443 }]'
     k  -n kube-system patch deployments metrics-server --type=json -p="$PATCH"    
 fi
 
-# kops 1-22 installs an older metrics server than we ship with eksd 1.22 along with an older clusterrole def.  The 0.6 version of metrics requires a slightly different rbac setup
```

Comment:
We might want to leave this comment as a clue for the next time we update kops to revisit this.

Reply:
Added back and updated to fit the change proposed in this PR